packages/grafana-ui/src/components/BgContainer/BgContainer.tsx

```diff
@@ -0,0 +1 @@
+export type BgColor = 'canvas' | 'primary';
```

Comment:
I was wondering what the thinking is behind defining two separate types (PluginPageBackground and BgColor) when they both represent the same strings. I’d love to understand the design choice a bit more.